Cargo.toml

```diff
@@ -14,6 +14,11 @@ keywords = ["crc", "crc16", "crc32", "crc64", "hash"]
 categories = ["algorithms", "no-std"]
 edition = "2018"
 
+[features]
+notable-defaults = []
```

Comment:
[nit] On naming, wdyt about "prefer-low-memory" and "prefer-fast" instead for "notable-defaults" and "slice16-defaults" respectively? They might be easier to understand for some users since it maps closer to intent.

Reply:
I'd expect the people that care about any of this to also care about the specifics i.e. `how much memory is "low memory" exactly` at which point they also need to read about the different implementations. To then name the features after these implementations seems logical to me

Reply:
I'll have another think about the naming

Reply:
I came up with 

* no-table-memory-restrictions
* bytewise-memory-restrictions
* slice16-memory-restrictions

I think this captures the aspect of the additional restrictions one is imposing by activating the features, hopefully also conveying that those are additive.

Reply:
Ah, that's a good idea! I've also seen "limit" used in this context. Wdyt about using "mem-limit" or "memory-limit" as the suffix? Has the advantage that it's shorter. Sorry for being so picky on the naming :)

Reply:
> Sorry for being so picky on the naming :)

No worries, naming are important and one of the hardest things to get right :)

"Limit" seems like a fine choice!